src/consensus/raft/raft.h

```diff
@@ -110,7 +101,7 @@ namespace raft
     // entries
     bool public_only = false;
 
-    // In recovery mode, while a follower is reading the private ledger, no
+    // In recovery mode, while a backup is reading the private ledger, no
```

Comment:
I think the renames in this file, especially in the comments, and ideally in variable names, should be undone. The only necessary changes are to the enum values we compare against, and the `Consensus` method names.